modules/understanding-node-cves-scanner-v4.adoc

```diff
@@ -0,0 +1,61 @@
+For this version:
+
+* StackRox Scanner reports 49 CVEs.
+* Scanner V4 reports 389 CVEs.
+
+The breakdown is as follows:
+
+* 1 CVE is reported only by the StackRox Scanner.
+* 48 CVEs are reported by both scanners.
+* 341 CVEs are reported only by Scanner V4.
+
+=== CVEs reported only by the StackRox Scanner
+
+The single CVE reported exclusively by StackRox Scanner was a false positive. CVE-2022-4122 was flagged for the `podman` package in version `5:5.2.2-1.rhaos4.17.el9.x86_64`. However, a manual review of VEX data from link:https://access.redhat.com/errata/RHSA-2024:9102[RHSA-2024:9102] indicated that this vulnerability was fixed in version `5:5.2.2-1.el9`. Therefore, the package version scanned was the first to contain the fix and was no longer affected.
+
+=== CVEs reported only by Scanner V4
+
+We randomly selected 10 CVEs from the 341 unique to Scanner V4 and conducted a detailed analysis using VEX data. The vulnerabilities fell into two categories:
+
+. Affected packages with a fine-grained status indicating that no fix is planned.
+. Affected packages with no additional status details regarding a fix.
+
+For example:
+
+* The `git-core` package (version `2.43.5-1.el9_4`) was flagged for CVE-2024-50349 (link:https://access.redhat.com/security/cve/cve-2024-50349[VEX data]), marked as "Affected" with a fine-grained status of "Fix deferred." This means a fix is not guaranteed due to higher-priority development work. The package is affected by three CVEs in total.
+
+* The `vim-minimal` package (version `2:8.2.2637-20.el9_1`) was flagged for 109 CVEs, 108 of which have low CVSS scores. Most are marked as "Affected" with a fine-grained status of "Will not fix."
+
+* The `krb5-libs` package (version `1.21.1-2.el9_4.1`) was flagged for CVE-2025-24528 (link:https://access.redhat.com/security/cve/cve-2025-24528[VEX data]), but no fine-grained status was available. Given that this CVE was recently discovered (2025), its status may be updated soon.
+
+=== CVEs reported by both scanners
+
+This category aligns with expectations. We manually verified three randomly selected packages, finding that OVAL2 data (used in the StackRox Scanner) and VEX data (used in Scanner V4) provided consistent explanations for the detected CVEs. In some cases, CVSS scores differed slightly, which is expected due to variations in VEX publisher data.
+
+== Recommendation
+
+We recommend manually verifying the fine-grained statuses of vulnerabilities in node host components that are critical to your environment using publicly available VEX data. VEX data is accessible in both human-readable and machine-readable formats. More information on interpreting VEX data is available in this link:https://www.redhat.com/en/blog/recent-improvements-red-hat-enterprise-linux-coreos-security-data[Red Hat Blog].
```

Comment:
```suggestion
As a best practice, verify the fine-grained statuses of vulnerabilities in node host components that are critical to your environment using publicly available VEX data. VEX data is accessible in both human-readable and machine-readable formats. For more information about interpreting VEX data, visit link:https://www.redhat.com/en/blog/recent-improvements-red-hat-enterprise-linux-coreos-security-data[Recent improvements in Red Hat Enterprise Linux CoreOS security data].
```
We are supposed to [avoid using "recommend"](https://redhat-documentation.github.io/supplementary-style-guide/#recommend) - tried to rewrite slightly.